customcontext/context.cpp

```diff
@@ -302,15 +262,25 @@ void Context::renderContextInitialized(QSGRenderContext *ctx)
 
 #endif // CUSTOMCONTEXT_HYBRISTEXTURE
 
-    QSGContext::renderContextInitialized(ctx);
+    CONTEXT_CLASS_BASE::renderContextInitialized(ctx);
 }
 
 
 
 
 
-void CONTEXT_CLASS::initialize(QOpenGLContext *context)
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+void RENDER_CONTEXT_CLASS::initialize(const InitParams *inCtx)
 {
+#elif QT_VERSION >= 0x050800
+void RENDER_CONTEXT_CLASS::initialize(void *inCtx)
+{
+    QOpenGLContext *context = static_cast<QOpenGLContext*>(inCtx);
+#else
+void RENDER_CONTEXT_CLASS::initialize(QOpenGLContext *inCtx)
```

Comment:
```
In file included from context.cpp:42:
context.h:70:30: error: no declaration matches ‘void CustomContext::RenderContext::initialize(QOpenGLContext*)’
 #define RENDER_CONTEXT_CLASS RenderContext
                              ^~~~~~~~~~~~~
context.cpp:280:6: note: in expansion of macro ‘RENDER_CONTEXT_CLASS’
 void RENDER_CONTEXT_CLASS::initialize(QOpenGLContext *inCtx)
      ^~~~~~~~~~~~~~~~~~~~
context.h:96:10: note: candidate is: ‘void CustomContext::RenderContext::initialize(void*)’
     void initialize(void *context);
          ^~~~~~~~~~
context.h:85:7: note: ‘class CustomContext::RenderContext’ defined here
 class RenderContext : public RENDER_CONTEXT_CLASS_BASE
       ^~~~~~~~~~~~~
make[1]: *** [Makefile:478: context.o] Error 1

```

Reply:
Probably fixed with https://github.com/sailfishos/qtscenegraph-adaptation/pull/1/commits/a70e9a9e883cd0e6e989e18462b33d356ad395ce but I only have 5.15 here to test